5g-control-plane/values.yaml

```diff
@@ -7,6 +7,7 @@ images:
   repository: "" #default docker hub
   tags:
     init: omecproject/pod-init:rel-1.1.2
+    attestation: storytel/alpine-bash-curl:latest
```

Comment:
I think we should find a better image for this, it seems unmaintained, and the organization is unknown to me.